ui/lib/pki/addon/components/pki-issuer-cross-sign.hbs

```diff
@@ -0,0 +1,82 @@
+<h3 for="intermediates to cross-sign" class="title is-4">Intermediates</h3>
```

Comment:
Curious if the `for` attribute is needed here? Typically it would be used on a `label` element and reference the id of an `input`.

Reply:
oh this `<h3>` had been a label and I changed the HTML tag - good catch! I can remove it :)